kafka-ui-react-app/src/components/Schemas/Edit/Edit.tsx

```diff
@@ -162,8 +160,11 @@ const Edit = ({
                 <Controller
                   control={control}
                   name="newSchema"
-                  disabled={isSubmitting}
-                  render={({ name, onChange }) => (
+                  render={({
+                    field: { name, onChange },
+                  }: {
+                    field: { name: string; onChange: () => void };
+                  }) => (
```

Comment:
Same here: no need to describe `field`. 
```suggestion
                  render={({ field: { name, onChange } }) => (
```